### PR TITLE
Allow designer to show tables from other DB

### PR DESCRIPTION
**Pull request by Achilles-96**
Signed-off-by: Raghuram Vadapalli <raghuram.4350@gmail.com>

Before submitting pull request, please check that every commit:

- [x] Has proper Signed-Off-By
- [x] Has commit message which describes it
- [x] Is needed on it's own, if you have just minor fixes to previous commits, you can squash them
- [ ] Any new functionality is covered by tests

### CODE REVIEW

**Comment by ibennetch** [comment]
This functions well except I don't seem to be able to save the layout with the other database's table showing. I can save and open the view, but the other table isn't shown.

**Comment by ibennetch** [comment]
Oh, right, we were talking about that...it's just hard for me to associate that conversation with this pull request, it's like a different context and I just missing making the connection.

File: libraries/pmd_common.php [hunk 1]
```diff
@@ -566,8 +566,8 @@ function PMA_addNewRelation($db, $T1, $F1, $T2, $F2, $on_delete, $on_update)
         $GLOBALS['dbi']->freeResult($result);
 
         $result = $GLOBALS['dbi']->query(
-            'SHOW INDEX FROM ' . PhpMyAdmin\Util::backquote($db)
-            . '.' . PhpMyAdmin\Util::backquote($T2) . ';'
+            'SHOW INDEX FROM ' . PMA\libraries\Util::backquote($DB2)
+            . '.' . PMA\libraries\Util::backquote($T2) . ';'
```
Comment (nijel) [inline libraries/pmd_common.php]: You're reverting namespace changes, this won't work...

**Comment by ibennetch** [comment]
Merged, thanks!

**Comment by ibennetch** [comment]
I made a few minor fixes, as we discussed, changing the double quotes to single quotes is 22ddc46e4d1291907ef2fc4c8d57f7199c9e7a63 and I removed a line that seems to be debugging code at 486daec6d1686bac3679bc0b537c6dd6ab7d7657.

I've also opened issues #13586 to add an icon and #13587 which are related to this feature.

Way to go. This is a very complex and fragile part of the code and you did a nice job.

**Comment by Achilles-96** [comment]
Thanks a lot, Isaac. :)

**Pull request merged**